website/content/en/docs/concepts/nodepools.md

```diff
@@ -178,13 +178,11 @@ status:
 Optional field that enables static capacity mode. When specified, the NodePool maintains a fixed number of nodes regardless of pod demand.
 
 **Static NodePool Constraints:**
-- Cannot be changed once set (NodePool cannot switch between static and dynamic modes)
-- Consolidation settings are ignored when replicas is specified
+- Cannot be removed once set (NodePool cannot switch between static and dynamic modes)
 - Only `limits.nodes` is allowed in limits section
 - `weight` field cannot be set
 - Nodes are not considered as consolidation candidates
```

Comment:
I'd still consider removing the "candidate" terminology since candidacy is an internal concept. I'd just word it as "Nodes are not considered for consolidation." FWIW I think it's pretty self-evident what candidacy means, I'd just avoid using these internal names in our docs.